client/src/components/settings/toggle-button-setting.tsx

```diff
@@ -35,7 +34,6 @@ export default function ToggleButtonSetting({
           aria-pressed={flag}
           {...(!flag && { onClick: toggleFlag })}
           value='1'
-          className='toggle-button-right'
```

Comment:
Silly question, probably, but doesn't this need to be ui-components Button, not just a button?